decidim-core/lib/decidim/api/types/translated_field_type.rb

```diff
@@ -38,10 +38,14 @@ def display_translations
       end
 
       def defined_translations
+        return {} unless object.is_a?(Hash)
```

Comment:
This is preventing a flaky or an error? Can you provide an example if this is the case?

Reply:
Ok! If we found out the error we can reintroduce the fix with a spec and do a `type: fix` PR